Kodelock.py

```diff
@@ -1,64 +1,64 @@
+#!/bin/python3
+
+from pathlib import Path
+
 from tkinter import *
-from PIL import ImageTk, Image 
+from PIL import ImageTk, Image
 from pynput import keyboard
 
+
 # Placeholder & Files
-app = Tk()
-codes = open("codes.txt", "r")
-CodeFace = Image.open("./imgs/Face.jpg")
-LeftArrow = Image.open("./imgs/LeftArr.png")
+codes = "codes.txt"
+code_list = Path(codes).read_text().splitlines()

```

Comment:
Simple 1 liner to read a file and split by new line. This will put all the codes into the 'code_list' list/array.